specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2024-07-01-preview/elasticsan.json

```diff
@@ -712,6 +742,21 @@
           },
           {
             "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "x-ms-access-soft-deleted-resources",
+            "description": "Optional, used to get soft deleted volume groups if set to true.",
```

Comment:
For get single VG, what's the difference of add this parameter or not? 
Does it mean:
If a VG is soft deleted, add the parameter will return the VG; without this parameter will report 404?

Reply:
Yes, that's right. If we try to get a soft deleted VG without this header, we will get 404

Reply:
What will happen when the VG is not deleted, and get it with this parameter?

Reply:
That too will return 404.

Reply:
This is not applicable now as we do not support GET for soft deleted volume groups now.